database/database_handler.py

```diff
@@ -0,0 +1,88 @@
+import os
+import subprocess
+import json
+
+from sqlalchemy import create_engine, Column, String, Boolean, Date, Integer, Float
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.exc import SQLAlchemyError
+
+from utils.general import LOGGER
```

Comment:
What is the reason for using a global logger object?

Reply:
This is the LOGGER that YOLOv5 uses. I want to use the same log library in the whole project, because I think this would make it easier for us to send the outputs to Azure logs